core/modules/field_ui/field_ui.theme.inc

```diff
@@ -102,11 +102,20 @@ function theme_field_ui_view_modes($variables) {
   $entity_type = $element['#entity_type'];
   $path = $element['#admin_path'] . '/display';
 
+  // Get the node type machine name.
+  $node_type = '';
+  foreach (node_type_get_types() as $node_type_name => $node_type) {
+    if ($node_type->name == $element['#bundle']) {
+      $node_type = $node_type_name;
+      break;
+    }
+  }
+
```

Comment:
This will only work for nodes. Please consider my PR here: https://github.com/BWPanda/backdrop/pull/14